Adds attack verb to baguette

RaveRadbury opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds attack verb to baguette: "touche'd" 
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The bike horn gets an attack verb so you know in chat that you've been honked and where you were honked. The mime has no such equivalent which is a shame because the baguette is totally slapstick. Now you can bash people on the head with the baguette and they'll know you're targeting their head and there will also be in-chat feedback for it.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
tweak: Baguettes now have an attack verb. Try hitting somebody with one!
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

RaveRadbury committed:
Adds attackby to Baguette

Comes with three possible verbs

necromanceranne commented:
Stale baguette time.

WACK WACK WACK

PKPenguin321 commented:
These are attack verbs, not attackby 😳

RaveRadbury commented:
> These are attack verbs, not attackby 😳

Hey thanks! My inexperience betrays me yet again!

cacogen commented:
make it make the attacking mob call the person a fagguette

ninjanomnom commented:
This is nonsensical to the point of potentially even being detrimental to understanding what's happening in character. This seems like a bad idea in poor taste.

cacogen commented:
on my quest to add faguette to the game i discovered the baguette has no attack message because its force is set to 0 so it doesn't do damage

RaveRadbury commented:
> This is nonsensical to the point of potentially even being detrimental to understanding what's happening in character. This seems like a bad idea in poor taste.

Okay, what if I just strip it down to "touche'd"? That's the fencing term for hitting your opponent with your sword. It's a french term, it plays into using the baguette like a prop, it communicates what it's doing. Does that strike a better balance?

cacogen commented on code/modules/food_and_drinks/food/snacks_bread.dm:
```diff
@@ -178,6 +178,7 @@
 	bitesize = 3
 	w_class = WEIGHT_CLASS_NORMAL
 	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT
+	attack_verb = list("hon hon'ed", "je ne sais quoi'ed", "touche'd")
```
```suggestion
	attack_verb = list("hon hon'ed", "je ne sais quoi'ed", "touche'd")
	force = 1
```

cacogen replied:
no force no attack message no attack verb

RaveRadbury replied:
I tested it without force and I was able to touche a monkey with a baguette. Are you saying that the receiver of the touche will not see this? Pretty sure the bike horn has a force of 0 and it's functional.

RaveRadbury committed:
Reduces attack verbs to "touche'd"

The French word for "touch". Used by the referee to declare that a touch has been made.

RaveRadbury commented:
Reduced the attack verb to "Touche'd". This conveys what is happening accurately, someone is being hit with a loaf of bread being wielded like a sword.

Jordie0608 commented:
I did fencing for 5 years so I approve of this.

cacogen commented:
I'm still convinced it doesn't give an attack message without force, based
on trying it out locally. Haven't looked at the code though.

On Thu, 12 Dec 2019 at 12:06 AM, Jordie <notifications@github.com> wrote:

> Merged #48187 <https://github.com/tgstation/tgstation/pull/48187> into
> master.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tgstation/tgstation/pull/48187?email_source=notifications&email_token=AF7NO6VOLWL7X7NSEJIRKYTQYDCUHA5CNFSM4JYSLGXKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOVNF3EFA#event-2873864724>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF7NO6QTMZZGXWBWJPLBBFLQYDCUHANCNFSM4JYSLGXA>
> .
>
-- 
Sent from my phone

Jordie0608 commented:
It does
https://github.com/tgstation/tgstation/blob/df4f50f6fa27f2e85fd52d8ba401febe027e535a/code/_onclick/item_attack.dm#L136-L141
